src/main/java/org/springframework/social/partnercenter/security/MicrosoftRedirectAccessGrant.java

```diff
@@ -0,0 +1,39 @@
+package org.springframework.social.partnercenter.security;
+
+import java.util.UUID;
+
+public class MicrosoftRedirectAccessGrant {
```

Comment:
Not sure about the name of this. 

Maybe DelegatedAccessGrant?

Reply:
👍